user-guide/modules/ROOT/pages/testing-debugging.adoc

```diff
@@ -0,0 +1,162 @@
+= Testing and Debugging
+:navtitle: Testing and Debugging
+
+Debugging code that uses Boost libraries follows the same general principles as debugging any other pass:[C++] code. 
```

Comment:
> follows the same general principles as debugging any other pass:[C++] code

This kind of statement might make some people want to skip to the next page. Since the page is also discussing Boost.Test, which is a variation of these general principles, maybe the text could mention why it's special?

Reply:
@alandefreitas Added more particulars to the intro, and mentioned boost.test is integrated into Visual Studio.

Reply:
Great. You just need to squash these commits and I think this is good to go.

Reply:
@alandefreitas squashed

Reply:
There are still two commits in the PR. Maybe you didn't force push to your `develop` branch.